azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/compute/AzureComputeServiceAdapter.java

```diff
@@ -176,6 +185,16 @@ public boolean apply(final String name) {
             hwProfile.location = location.name();
             hwProfiles.add(hwProfile);
          }
+*/
+         VMHardware hwProfile = new VMHardware();
+         hwProfile.name = "Standard_A5";
+         hwProfile.numberOfCores = 2;
+         hwProfile.osDiskSizeInMB = 1047552;
+         hwProfile.resourceDiskSizeInMB = 138240;
+         hwProfile.memoryInMB = 14336;
+         hwProfile.maxDataDiskCount = 4;
+         hwProfile.location = "westus";
+         hwProfiles.add(hwProfile);
```

Comment:
We need to remove the hardcoded hardware profiles before merging. We don't want to restrict users just to this configuration.